cooperator/views/subscription_request_view.xml

```diff
@@ -129,7 +129,7 @@ SPDX-License-Identifier: AGPL-3.0-or-later
                             />
                             <field name="firstname" />
                             <field name="lastname" />
-                            <field name="birthdate" />
+                            <field name="birthdate" required="1" />
```

Comment:
is there a reason to use `1` instead of `True`? i know that both work and both can be found in odoo’s views (even twice as many `1` than `True`), but i think it’s better to use the proper type (as internally it’s a boolean property).

Reply:
Yes I think you're right. The warning would make sense, since there should be birthdate on cooperators. 
I fixed it. Also fixed the demo data.